app/controllers/web_api_controller.rb

```diff
@@ -0,0 +1,12 @@
+# This controller provides UI access to investigate and
+# browse the RESTful API endpoints using a Swagger UI
+# and the automapping from the Swagger data at /api/swagger.json
+#
+# Swagger UI migrated from original repository at https://github.com/swagger-api/swagger-ui
+
+class WebApiController < ApplicationController
```

Comment:
I'm fine with this being a controller method, but if you're interested in a static file, you can actually make a folder structure like `/public/api/index.html` and put a static HTML file there. But you wouldn't get the default layout (header, footer) if you did, not sure if that's a pro or a con. Just a note in case that's simpler?